openfe/tests/protocols/test_openmmutils.py

```diff
@@ -244,7 +245,7 @@ def reporter(self):
         finally:
             r.close()
 
-    @pytest.fixture()
+    @pytest.fixture(scope='class')
```

Comment:
same question as above

Reply:
(for the future) this one really benefits from a session scope, we'll try to leave it here for now.